requirements.txt

```
@@ -1,23 +1,24 @@
-attrs==23.1.0
+attrs==24.2.0
 aws-requests-auth==0.4.3
-bdbag==1.6.3
-boto3==1.28.63  # match this with the version of the `boto3-stubs` dev dependency
-botocore==1.31.63
+bdbag==1.7.3
+boto3==1.35.48  # match this with the version of the `boto3-stubs` dev dependency
+botocore==1.35.48
 chevron==0.14.0
-deprecated==1.2.13
-elasticsearch-dsl==7.4.0
-elasticsearch==7.17.9
-fastavro==1.8.2
+deprecated==1.2.14
+elasticsearch==7.17.12
+elasticsearch-dsl==7.4.1
+fastavro==1.9.7
 furl==2.1.3
-google-auth[pyopenssl]==2.23.3
-google-cloud-bigquery==3.12.0
-google-api-core==2.12.0
-http-message-signatures==0.4.4
+google-api-core==2.21.0
+google-auth[pyopenssl]==2.34.0
+# Updating the following package beyond 3.13.x > breaks test & production code
+google-cloud-bigquery==3.13.0
```

Can't have comment lines like this. Each line needs to be a dependency. If there is a comment, it needs to be at the end of a dependency line. Please create a ticket and 

```suggestion
google-cloud-bigquery==3.13.0  # Can't go higher, due to <insert ticket reference here>
```

The ticket should have a reproduction indicating what breaks if we were to go higher. The title of the commit should refer to #6511 and the new ticket.

---

Understood. But just to clarify, I was following the directive as prescribed ([last sentence](https://github.com/DataBiosphere/azul/issues/6511#issuecomment-2299395540)).